independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "You can use Electricity Maps API to get access to information about where the electricity in a specific area comes from, how much carbon was emitted to produce it, and to get historical data, live data and forecasts.",
```

Comment:
Hi @VitaliiSorokin, Please remove the word API from the description. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)